packages/node_modules/cerebral/index.d.ts

```diff
@@ -79,7 +79,7 @@ export interface ControllerClass extends FunctionTree {
   getModel(): StateModel
   getState(path?: string): any
   runSignal(name: string, signal: SignalChain, payload: any): void
-  getSignal(path: string): Signal
+  getSignal<T=any>(path: string): Signal<T>
```

Comment:
What I don'like here about typing systems like ts and flow is that is could be treated as breaking change :(
But bump major version on each such change is overkill